src/components/Post/Post.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+interface PostsProps {
+  user: string;
+  title: string;
+  text: string;
+  dateAndTime: string;
+}
+
+function getProperDate(dateString: string): string {
```

Comment:
define it as arrow function. Try to keep the same coding style for all parts of application.